scripting/get5/matchconfig.sp

```diff
@@ -257,7 +257,8 @@ static void MatchConfigFail(const char[] reason, any...) {
 }
 
 stock bool LoadMatchFromUrl(const char[] url, ArrayList paramNames = null,
-                            ArrayList paramValues = null) {
+                            ArrayList paramValues = null, ArrayList headerNames = null,
+                            ArrayList headerValues = null) {
```

Comment:
The header name and value should not be arraylists, as there is always only one string in each. It should be `char[]` buffers.

Reply:
Unless you intentionally want these to be able to accept multiple values.

Reply:
I've done that intentionally, because for me it made sense to implement this with the most flexibility in mind. 
Also since the params are also implemented like that but they are not used from the command line function, the headers can also be used in the same way if you use the natives from somewhere. Or do you see any problem with this implementation?